CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/parameters/types/FileParameterType.java

```diff
@@ -343,7 +352,16 @@ public void enableAcceptAllFileFilter() {
 
         @Override
         public String validateString(final String s) {
-            return null;
+            File validationFile = new File(s);
+            //Accepted Conditions:
+            //The current file is a directory
+            //The current file has no parent
+            //the current file has a prent that is a directory
+            if (validationFile.isDirectory() || (!validationFile.isDirectory() && validationFile.getParentFile() != null && validationFile.getParentFile().exists())){
```

Comment:
The first bit of the clause after || is redundant as this is implied by the first condition